pkg/rpctype/rpctype.go

```diff
@@ -27,6 +27,7 @@ type RPCCandidate struct {
 type ConnectArgs struct {
 	Name        string
 	MachineInfo []byte
+	Modules     []host.KernelModule
```

Comment:
Passing them here is interesting option. I think it's simpler than what I proposed before.